kickoff/jsonexportl10n.py

```diff
@@ -36,7 +36,7 @@ def l10nExport(releaseName):
 def _aggregateBetaLocales(releaseTable, releaseName):
     beta_prefix = releaseName.replace("beta", "b")
 
-    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix))
+    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix)).filter(releaseTable._shippedAt != None)
```

Comment:
Additionally -- you can use ```filter(*args)``` instead of ```filter(arg).filter(arg2)``` if you want. See: [Official Docs](http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.filter)